core/metrics/src/main/scala/org/typelevel/otel4s/metrics/BatchCallback.scala

```diff
@@ -286,6 +288,14 @@ trait BatchCallback[F[_]] {
         )
       }
 
+  /** Modify the context `F` using an implicit [[KindTransformer]] from `F` to `G`.
+    */
+  def mapK[G[_]: MonadCancelThrow](implicit
+      mct: MonadCancelThrow[F],
+      kt: KindTransformer[F, G],
+      tk: KindTransformer[G, F]
```

Comment:
just some name bikeshedding:

- monadic constraints tend to just share the name of the HKT (idk why, but it's conventional
- `kt` is just the first letters of 'Kind' and 'Transformer', so `tk` doesn't make a lot of sense as a name

```suggestion
      F: MonadCancelThrow[F],
      ktFG: KindTransformer[F, G],
      ktGF: KindTransformer[G, F]
```